src/main/java/org/popcraft/chunky/watchdog/AbstractGenerationWatchdog.java

```diff
@@ -0,0 +1,9 @@
+package org.popcraft.chunky.watchdog;
+
+public abstract class AbstractGenerationWatchdog {

```

Comment:
Probably meant to make this an interface since there are no methods being implemented here